frontend/packages/vkt/src/utils/serialization.ts

```diff
@@ -56,10 +56,14 @@ export class SerializationUtils {
   static deserializeClerkEnrollment(
     enrollment: ClerkExamEventEnrollmentResponse
   ) {
+    const previousEnrollmentDate = enrollment.previousEnrollmentDate
```

Comment:
Ilman tätä muutosta tuli näkymään `12.12.2022` / meneillään oleva päivä aikaisempana osallistumispäivämääränä vaikka pvm. olisi ollut kannassa null

Reply:
Aivan, tämähän löyty dayjs docseista kun katsoi:

`Day.js treats dayjs(undefined) as dayjs() due to that function parameters default to undefined when not passed in.`

Reply:
Aivan, tämähän löyty dayjs docseista ihan: ´Day.js treats dayjs(undefined) as dayjs() due to that function parameters default to undefined when not passed in.`